### PR TITLE
Add Swedish translations

### PR DESCRIPTION
**Pull request by AlexGustafsson**

### CODE REVIEW

File: Multiplatform/Localizable.xcstrings [hunk 1]
```diff
@@ -210,6 +282,7 @@
       }
     },
     "%lld episodes.unplayed" : {
+      "comment" : "Could differ depending on if there's one unplayed or many unplayed.",
```
Comment (rasmuslos) [inline Multiplatform/Localizable.xcstrings:285]: Good idea, could you do it and add the translation?

**Comment by rasmuslos** [comment]
Thanks a lot for your effort. You picked a great time, too. Version `2.5.0` will be released soon and the translations should no longer change. I updated the contribution guide a bit, it now mentions the actual second file `InfoPlist.xcstrings`, located at the same place. But there is only one translation in there, so don't worry.

You also have to add Swedish as a supported language to the project:

Bildschirmfoto 2024-11-17 um 09 51 24

File: Multiplatform/Localizable.xcstrings (hunk 1 above)
Comment (AlexGustafsson) [reply Multiplatform/Localizable.xcstrings]: Should be fixed now.

**Comment by rasmuslos** [comment]
I will add the `AppIntentVocabulary` to the instructions, too. I am not sure if the Intents file is actually still in use, but there is no harm in translating it, Thanks! But the Siri usage description is still missing in `InfoPlist.xcstrings`, next to the `Localizable.xcstrings`.

Looks great otherwise, Thanks!

**Comment by AlexGustafsson** [comment]
@rasmuslos Sorry. Fixed.

**Comment by rasmuslos** [comment]
Nice work! Thank you 👍

**Comment by rasmuslos** [comment]
2.5.0 will be released soon, but if you want to you send me an E-Mail to git@rfk.io from the address you use for the AppStore which I will use to add you to the TeatFlight beta.

**Pull request merged**